pkg/ipmi/constants.go

```diff
@@ -37,6 +37,11 @@ const (
 	// LAN Device Commands
 	BMC_GET_LAN_CONFIG Command = 0x02
 
+	// Boot Time Measurement Commands
+	BMC_SEND_REBOOT_CHECKPOINT          Command = 0x12
```

Comment:
You can define these in an internal to google file, if these are not IPMI standard. We can forego upstreaming for stuff that's not useful to the outside world. (I'm making an assumption here, lmk if I'm wrong)

Reply:
That is a fair point. I saw these IPMI commands were made public in openbmc (https://github.com/openbmc/google-ipmi-sys/blob/master/commands.hpp), but now realizing it is a google-specific branch.

Reply:
@hugelgupf @rjoleary : Sure let me move these definitions to internal to google file, thanks.